investigation-service/src/main/resources/application.yaml

```diff
@@ -9,7 +9,6 @@ spring:
       topic-name-observation: nrt_investigation_observation
       topic-name-notifications: nrt_notifications
     dlq:
-      topic-name-dlq: nrt_investigation_dlq
       retry-suffix: _retry
       dlq-suffix: _dlt
```

Comment:
do we want to change here too from underscore to dash (`-retry` and `-dlt`)

Reply:
those suffixes applied to `nbs_` topics only in the pre-processing